src/Tribe/Admin/Notices.php

```diff
@@ -329,4 +330,60 @@ function maybe_display_paystack_notice() {
 		);
 
 	}
+
+	/**
+	 * Display AR Page notice for FSE theme detected.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function maybe_display_fse_ar_page_notice() {
+		// Bail if we aren't in Tickets > Settings.
+		if ( \Tribe\Tickets\Admin\Settings::$settings_page_id !== tribe_get_request_var( 'page' ) ) {
+			return;
+		}
+
+		// Bail if Tickets Plus is not active.
+		if ( ! class_exists( 'Tribe__Tickets_Plus__Main' ) ) {
+			return;
+		}
+
+		// Bail if not using FSE theme.
+		if ( ! is_readable( get_stylesheet_directory() . '/templates/index.html' ) ) {
+			return;
+		}
+
+		// Bail if the attendee registration page is already set.
+		$id = Tribe__Settings_Manager::get_option( 'ticket-attendee-page-id', false );
+		if ( ! empty( $id ) ) {
+			return;
+		}
```

Comment:
Are we sure we don't need additional validation here ?

Like that the saved id still exists ? Or if it is still published ? etc